app/pages/home-common/featured-project.jsx

```diff
@@ -1,32 +1,42 @@
-/*
-Featured Project Component
-==========================
-* This component highlights specific projects on the home page.
-* NOT to be confused with <HomePagePromoted />
-* Content is hardcoded for the moment; improvements on this are welcome.
-* Originally created to highlight 2017 Mar/Apr Stargazing Live projects.
-* Design: @beckyrother; code: @shaunanoordin; documented/updated: 20171023
-* Compress and resize any static images before uploading and deploying!
-*/
-
+import PropTypes from 'prop-types';
 import React from 'react';
 import { Link } from 'react-router';
 import Thumbnail from '../../components/thumbnail';
 
-const FeaturedProject = (() => (
-  <section className="home-featured">
-    <h1 className="secondary-kicker">Featured Project</h1>
-    <div className="home-featured-images">
-      <img
-        alt="SCOTUS Notes"
-        src="/assets/featured-projects/featured-project-20180223-scotus-notes.jpg"
-      />
-    </div>
-    <h2 className="secondary-headline">SCOTUS Notes</h2>
-    <p className="display-body">Behind the scenes at Supreme Court Conference</p>
-    <Link to="projects/zooniverse/scotus-notes-behind-the-scenes-at-supreme-court-conference">View Project!</Link>
-  </section>
-  )
-);
+function FeaturedProject({ project }) {
+  if (project) {
+    const featuredImage = ((project.avatar_src && `//${project.avatar_src}`) || (project.links.background && project.links.background.href));
```

Comment:
`project.links.background.href` isn't an image URL, it's the API URL for the linked `background` resource. If you want to use the background image, you have to request the background from Panoptes, then use `background.src`.

Reply:
The project background was initially intended to be a fallback when avatar wasn't available. However, since now I'm filtering through `launch_approved` projects now, there no need for a fallback. I'll remove that bit.

Reply:
Actually, having an avatar isn't a requirement for launching a project, is it? For The Plastic Tide doesn't have one. Should we make it a requirement? Now I'm not sure which image would be safe to use for the featured project. Any thoughts?

Reply:
The Plastic Tide does have an avatar. It's the image that loads on https://featured-project-toggle.pfe-preview.zooniverse.org/?env=production after the initial request to thumbnails.zooniverse.org fails.

Reply:
Sorry I got that wrong.